.env.template

```diff
@@ -5,8 +5,10 @@ USER=
 PGHOST=
 POSTGRES_PASSWORD=
 PGPASSWORD=
+PGDATA=
 OPENAI_API_KEY=
-AZURE_OPENAI_SERVICE=
+OPENAI_ENDPOINT=
+OPENAI_API_ENGINE=
 LOUIS_SCHEMA=
 DB_SERVER_CONTAINER_NAME=
-PGDATA=
+FINESSE_WEIGHTS=
```

Comment:
the red marker at the end of files here indicates that you haven't finished the file with a newline.

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

@redat97 document this in our developer guidance somewhere with tooling so that VSC ensures the file ends with newline for all developers. I don't want to have to manually point it out in any other review.

Reply:
I can just implement the workflow as well no? would make it easier

Reply:
I corrected this